packages/SwingSet/src/kernel/vatManager/manager-local.js

```diff
@@ -84,6 +84,7 @@ export function makeLocalVatManagerFactory(tools) {
   async function createFromBundle(vatID, bundle, managerOptions) {
     const {
       metered = false,
+      enableDisavow = false,
```

Comment:
All these different kinds of manager & worker setups need to have the same stuff threaded through them, which makes changes to the configuration options tedious.  We've got enough parameters at this point that I'm wondering if this is the API we really want.

Reply:
Yeah, it's due for a cleanup.. part of the problem is the incomplete overlap between the `managerOptions` that could be passed in, and the options that are going into `makeLiveSlots`. Certainly `makeLiveSlots` now has enough positional parameters to justify creating an options bag or something, but I don't know what the cleanest pattern would be.